release-1.8/release_notes_draft.md

```diff
@@ -478,16 +478,50 @@ kind.
 
 ### API Machinery
 
+
+#### kube-apiserver
+* Fixes an issue with APIService auto-registration affecting rolling HA apiserver restarts that add or remove API groups being served.([#51921](https://github.com/kubernetes/kubernetes/pull/51921))
+
+* [Alpha] The Kubernetes API server now supports the ability to break large LIST calls into multiple smaller chunks.  A client can specify a limit to the number of results to return, and if more results exist a token will be returned that allows the client to continue the previous list call repeatedly until all results are retrieved.  The resulting list is identical to a list call that does not perform chunking thanks to capabilities provided by etcd3.  This allows the server to use less memory and CPU responding with very large lists.  This feature is gated as APIListChunking and is not enabled by default.  The 1.9 release will begin using this by default from all informers.([#48921](https://github.com/kubernetes/kubernetes/pull/48921))
+
+* Ignore pods marked for deletion that exceed their grace period in ResourceQuota([#46542](https://github.com/kubernetes/kubernetes/pull/46542))
```

Comment:
Ignore the pods marked for deletion that exceed their grace period in ResourceQuota.